packages/blocks/src/__internal__/utils/common-operations.ts

```diff
@@ -42,6 +42,11 @@ export function isCollapsedAtBlockStart(vEditor: VEditor) {
   return vRange?.index === 0 && vRange?.length === 0;
 }
 
+export function isCollapsedAtBlockEnd(vEditor: VEditor) {
+  const vRange = vEditor.getVRange();
+  return vRange?.index === vEditor.yText.length;
```

Comment:
Missing condition `vRange.length === 0` check to make sure range is collapsed

Reply:
Sorry, I don't understand **collapsed** here. This code means checking the current cursor at the end of this line, in this case, ` vRange.length === 0` is always `true`.
If you want to help, PR is welcome!

Reply:
collapsed means that the range is a cursor, otherwise it is not.